packages/hash/api/src/model/ontology/entity-type.model.ts

```diff
@@ -285,19 +267,24 @@ export default class {
   /**
    * Update an entity type.
    *
-   * @param params.accountId the accountId of the account making the update
    * @param params.schema an `EntityType`
    */
   async update(
     graphApi: GraphApi,
     params: {
-      accountId: string;
       schema: Omit<EntityType, "$id">;
     },
   ): Promise<EntityTypeModel> {
-    const { accountId, schema } = params;
+    const { schema } = params;
     const updateArguments: UpdateEntityTypeRequest = {
-      accountId,
+      /**
+       * @todo: let caller update who owns the type, or create new method dedicated to changing the owner of the type
+       * @see https://app.asana.com/0/1202805690238892/1203063463721791/f
```

Comment:
As before with the `todo`s, this seems to be the same as the todo underneath in the comment. Probably the case for all ontology type `update` methods

Reply:
Yea, thanks for spotting - have fixed the links in https://github.com/hashintel/hash/pull/1129/commits/5b6dd0e31c8095aab8984efaaa830dd733e1d081